README.rst

```diff
@@ -127,6 +127,40 @@ To try out changes before applying them, use the ``--dry-run`` option. For insta
 
    ./manage.py lms forum_delete_course_from_mongodb all --dry-run
 
+Building MongoDB Indexes
+------------------------
+
+To optimize MongoDB query performance, it is crucial to create database indexes. The command will create or update indexes and skip them if they already exist.
+
+To create or update MongoDB indexes, execute the following command:
+
+    ./manage.py lms forum_create_mongodb_indexes
+
+Search indicies
+---------------
+
+Based on your search backend i.e Elasticsearch or Meilisearch, the commands will populate search indexes.
+
+Initialize Forum Indices
+-------------------------
+
+The `initialize_forum_indices` command used to initialize Elasticsearch indices for the forum. It allows you to force the creation of new indices even if they already exist.
```

Comment:
Verbatim code is surrounded in double backticks:

    The ``initialize_forum_indices`` command...

(same below)